omnibus-sepolia-carina.toml

```diff
@@ -0,0 +1,436 @@
+
+# --- Configure collateral for BFP --- #
+#
+#
+# TODO figure out how to get sUSD address
+[invoke.set_margin_collateral_configuration]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setMarginCollateralConfiguration"
+args = [
+    [ # collateral addresses
+        "<%= settings.weth_address %>",
+        "<%= settings.wsteth_address %>",
+    ],
+    [ # oracle node ids
+        "<%= extras.eth_oracle_id %>",
+        "<%= extras.wsteth_oracle_node_id %>",
+    ],
+    [ # maxAllowances
+        "<%= settings.weth_max_collateral %>",
+        "<%= settings.wsteth_max_collateral %>",
+    ],
+    [ # skewScales
+        "<%= settings.weth_skew_scale %>",
+        "<%= settings.wsteth_skew_scale %>",
+    ],
+    [ # rewardDistributors
+        "<%= extras.weth_rewards_distributor %>",
+        "<%= extras.wsteth_rewards_distributor %>",
+    ],
+]
+
+# --- Configure Core --- #
+#
+#
+
+# Add all perp market collateral as core collateral.
+# ETH and SNX have already been added from imports.
+#
+#
+#
+[invoke.configureWstethCollateral]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "configureCollateral"
+
+[[invoke.configureWstethCollateral.args]]
+tokenAddress = "<%= settings.wsteth_address %>"
+oracleNodeId = "<%= extras.wsteth_oracle_node_id %>"
+issuanceRatioD18 = "<%= parseEther('3') %>"
+liquidationRatioD18 = "<%= parseEther('1.5') %>"
+liquidationRewardD18 = "<%= parseEther('0.01') %>"
+minDelegationD18 = "<%= parseEther('0.01') %>"
+depositingEnabled = false
+
+# --- Configure maximum market collateral in Core --- #
+#
+#
+
+[invoke.weth_maximum_market_collateral]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "configureMaximumMarketCollateral"
+args = [
+    "<%= extras.eth_market_id %>",
+    "<%= settings.weth_address %>",
+    "<%= settings.weth_max_collateral %>",
+]
+
+[invoke.wsteth_maximum_market_collateral]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "configureMaximumMarketCollateral"
+args = [
+    "<%= extras.eth_market_id %>",
+    "<%= settings.wsteth_address %>",
+    "<%= settings.wsteth_max_collateral %>",
+]
+
+# --- Register Reward Distributors with Core --- #
+#
+#
+
+[invoke.register_wsteth_rewards_distributor]
+target = ["system.CoreProxy"]
+fromCall.func = "getPoolOwner"
+fromCall.args = ["<%= settings.sc_pool_id %>"]
+func = "registerRewardsDistributor"
+args = [
+    "<%= settings.sc_pool_id %>",
+    "<%= settings.wsteth_address %>",
+    "<%= extras.wsteth_rewards_distributor %>",
+]
+
+[invoke.register_weth_rewards_distributor]
+target = ["system.CoreProxy"]
+fromCall.func = "getPoolOwner"
+fromCall.args = ["<%= settings.sc_pool_id %>"]
+func = "registerRewardsDistributor"
+args = [
+    "<%= settings.sc_pool_id %>",
+    "<%= settings.weth_address %>",
+    "<%= extras.weth_rewards_distributor %>",
+]
+
+# --- Configure SC pool after market creation--- #
+#
+#
+
+[invoke.set_pool_configuration_sc]
```

Comment:
Much easier without spot market 👍

Reply:
Yea we still might want some spot markets here though